parse_convert/modules/samtools.nf

```diff
@@ -19,4 +19,21 @@ process SamtoolsMergeBams{
     samtools merge -p -c -O bam ${sample_id}.merged.bam \$(find . -name '*.bam')
     samtools index ${sample_id}.merged.bam
```

Comment:
Should we remove indexing in `SamtoolsMergeBams`, since we now have indexing process `SamtoolsIndexBam`?
Also adjust input to single channel, include `val(fq_id)` in input channel?